packages/jellyfish-api-core/__tests__/category/wallet.test.ts

```diff
@@ -632,3 +632,185 @@ describe('masternode', () => {
     })
   })
 })
+
+describe('sendMany', () => {
+  // NOTE(sp): defid side(c++) does not have much tests for sendmany RPC atm.
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  // util functions NOTE(sp) : may be move to a common util file if possible ??
```

Comment:
yea we actually have it in `@defichain/testing`

Reply:
@surangap you can keep it in this file as it is.

for `api-core` we try not to add too many test utils boilerplate (files) to keep the test scoped down for brevity. For other packages, e.g. the more complex use cases we have and uses a lot of testing utils. @canonbrother is maintaining the testing utilities now.

Reply:
We trying to contain the code debt so that it doesn't spaghetti around each other.